BeSpoke/ContentView.swift

```diff
@@ -52,6 +52,36 @@ struct ContentView: View {
             should probably happen at some point eventually maybe if we ever work on this more
             """)
     ]
+    
+    let tags: [Tag] = [
+        Tag(
+            id: 1,
+            notes: [1, 2],
```

Comment:
I gotta review standards for nosql structures - iirc it's usually better to have a separate object for each tag instance e.g. this 'smiley' tag would have an object for note 1 and a separate object for note 2 - but not totally sure (also doesnt matter for right now either way). I think its generally to avoid having potentially infinitely long fields (since there could be **a lot** of notes with that tag)